<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>862</width>
    <height>818</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>sigrok-qt</string>
  </property>
  <property name="windowIcon">
   <iconset resource="sigrok-qt.qrc">
    <normaloff>:/icons/sigrok-logo-notext.png</normaloff>:/icons/sigrok-logo-notext.png</iconset>
  </property>
  <widget class="QWidget" name="centralWidget">
   <layout class="QVBoxLayout" name="verticalLayout">
    <item>
     <widget class="QWidget" name="mainWidget" native="true">
      <layout class="QVBoxLayout" name="verticalLayout_2">
       <property name="margin">
        <number>0</number>
       </property>
       <item>
        <widget class="QTabWidget" name="tabWidget">
         <property name="currentIndex">
          <number>1</number>
         </property>
         <widget class="QWidget" name="tabLA">
          <attribute name="title">
           <string>Logic analyzer</string>
          </attribute>
          <widget class="QWidget" name="layoutWidget">
           <property name="geometry">
            <rect>
             <x>6</x>
             <y>10</y>
             <width>821</width>
             <height>61</height>
            </rect>
           </property>
           <layout class="QGridLayout" name="gridLayout">
            <property name="leftMargin">
             <number>2</number>
            </property>
            <property name="topMargin">
             <number>1</number>
            </property>
            <property name="rightMargin">
             <number>1</number>
            </property>
            <property name="bottomMargin">
             <number>1</number>
            </property>
            <item row="0" column="0">
             <widget class="QComboBox" name="comboBoxLA">
              <item>
               <property name="text">
                <string>No LA detected</string>
               </property>
              </item>
             </widget>
            </item>
            <item row="4" column="0">
             <widget class="QLabel" name="labelSampleStart">
              <property name="enabled">
               <bool>false</bool>
              </property>
              <property name="text">
               <string>Start sample:</string>
              </property>
             </widget>
            </item>
            <item row="0" column="1">
             <widget class="QLabel" name="labelChannels">
              <property name="enabled">
               <bool>false</bool>
              </property>
              <property name="text">
               <string>Channels:</string>
              </property>
             </widget>
            </item>
            <item row="0" column="2">
             <widget class="QComboBox" name="comboBoxSampleRate">
              <property name="enabled">
               <bool>false</bool>
              </property>
             </widget>
            </item>
            <item row="0" column="3">
             <widget class="QComboBox" name="comboBoxNumSamples">
              <property name="enabled">
               <bool>false</bool>
              </property>
             </widget>
            </item>
            <item row="4" column="1">
             <widget class="QLabel" name="labelSampleEnd">
              <property name="enabled">
               <bool>false</bool>
              </property>
              <property name="text">
               <string>End sample:</string>
              </property>
             </widget>
            </item>
            <item row="4" column="2">
             <widget class="QLabel" name="labelScaleFactor">
              <property name="enabled">
               <bool>false</bool>
              </property>
              <property name="text">
               <string>Scale factor:</string>
              </property>
             </widget>
            </item>
           </layout>
          </widget>
         </widget>
         <widget class="QWidget" name="tabLog">
          <attribute name="title">
           <string>Log</string>
          </attribute>
          <widget class="QPlainTextEdit" name="plainTextEdit">
           <property name="geometry">
            <rect>
             <x>10</x>
             <y>10</y>
             <width>821</width>
             <height>631</height>
            </rect>
           </property>
          </widget>
         </widget>
        </widget>
       </item>
      </layout>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menuBar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>862</width>
     <height>20</height>
    </rect>
   </property>
   <widget class="QMenu" name="menuFile">
    <property name="title">
     <string>&amp;File</string>
    </property>
    <addaction name="action_New"/>
    <addaction name="action_Open"/>
    <addaction name="action_Save_as"/>
    <addaction name="actionQuit"/>
   </widget>
   <widget class="QMenu" name="menuHelp">
    <property name="title">
     <string>&amp;Help</string>
    </property>
    <addaction name="actionAbout_Qt"/>
    <addaction name="actionAbout"/>
   </widget>
   <widget class="QMenu" name="menuHardware">
    <property name="title">
     <string>Device</string>
    </property>
    <addaction name="actionScan"/>
    <addaction name="action_Get_samples"/>
   </widget>
   <widget class="QMenu" name="menuEdit">
    <property name="title">
     <string>&amp;Edit</string>
    </property>
    <addaction name="actionPreferences"/>
   </widget>
   <widget class="QMenu" name="menuDecoders">
    <property name="title">
     <string>Decoders</string>
    </property>
    <addaction name="actionProtocol_decoder_stacks"/>
    <addaction name="actionConfigure"/>
   </widget>
   <addaction name="menuFile"/>
   <addaction name="menuHardware"/>
   <addaction name="menuEdit"/>
   <addaction name="menuDecoders"/>
   <addaction name="menuHelp"/>
  </widget>
  <widget class="QToolBar" name="mainToolBar">
   <attribute name="toolBarArea">
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
   <addaction name="action_New"/>
   <addaction name="action_Open"/>
   <addaction name="action_Save_as"/>
   <addaction name="actionPreferences"/>
   <addaction name="separator"/>
   <addaction name="actionScan"/>
   <addaction name="action_Get_samples"/>
   <addaction name="separator"/>
   <addaction name="actionQuit"/>
  </widget>
  <widget class="QStatusBar" name="statusBar"/>
  <action name="actionQuit">
   <property name="icon">
    <iconset resource="sigrok-qt.qrc">
     <normaloff>:/icons/system-log-out.png</normaloff>:/icons/system-log-out.png</iconset>
   </property>
   <property name="text">
    <string>E&amp;xit</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+Q</string>
   </property>
  </action>
  <action name="actionAbout">
   <property name="text">
    <string>&amp;About...</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+A</string>
   </property>
  </action>
  <action name="actionAbout_Qt">
   <property name="text">
    <string>About &amp;Qt...</string>
   </property>
  </action>
  <action name="actionPreferences">
   <property name="icon">
    <iconset resource="sigrok-qt.qrc">
     <normaloff>:/icons/preferences-system.png</normaloff>:/icons/preferences-system.png</iconset>
   </property>
   <property name="text">
    <string>&amp;Preferences</string>
   </property>
  </action>
  <action name="actionScan">
   <property name="icon">
    <iconset resource="sigrok-qt.qrc">
     <normaloff>:/icons/view-refresh.png</normaloff>:/icons/view-refresh.png</iconset>
   </property>
   <property name="text">
    <string>&amp;Scan</string>
   </property>
   <property name="toolTip">
    <string>Scan for supported logic analyzers</string>
   </property>
  </action>
  <action name="action_Open">
   <property name="icon">
    <iconset resource="sigrok-qt.qrc">
     <normaloff>:/icons/document-open.png</normaloff>:/icons/document-open.png</iconset>
   </property>
   <property name="text">
    <string>&amp;Open...</string>
   </property>
  </action>
  <action name="action_New">
   <property name="icon">
    <iconset resource="sigrok-qt.qrc">
     <normaloff>:/icons/document-new.png</normaloff>:/icons/document-new.png</iconset>
   </property>
   <property name="text">
    <string>&amp;New...</string>
   </property>
  </action>
  <action name="action_Save_as">
   <property name="enabled">
    <bool>false</bool>
   </property>
   <property name="icon">
    <iconset resource="sigrok-qt.qrc">
     <normaloff>:/icons/document-save-as.png</normaloff>:/icons/document-save-as.png</iconset>
   </property>
   <property name="text">
    <string>&amp;Save as...</string>
   </property>
  </action>
  <action name="action_Get_samples">
   <property name="enabled">
    <bool>false</bool>
   </property>
   <property name="icon">
    <iconset resource="sigrok-qt.qrc">
     <normaloff>:/icons/media-record.png</normaloff>:/icons/media-record.png</iconset>
   </property>
   <property name="text">
    <string>&amp;Get samples...</string>
   </property>
  </action>
  <action name="actionConfigure">
   <property name="text">
    <string>&amp;Configure</string>
   </property>
   <property name="toolTip">
    <string>Configure</string>
   </property>
  </action>
  <action name="actionProtocol_decoder_stacks">
   <property name="text">
    <string>Protocol decoder stacks</string>
   </property>
  </action>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources>
  <include location="sigrok-qt.qrc"/>
 </resources>
 <connections>
  <connection>
   <sender>actionQuit</sender>
   <signal>triggered()</signal>
   <receiver>MainWindow</receiver>
   <slot>close()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>-1</x>
     <y>-1</y>
    </hint>
    <hint type="destinationlabel">
     <x>299</x>
     <y>199</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
